service/telemetry/config.go

```diff
@@ -13,6 +13,14 @@ import (
 	"go.opentelemetry.io/collector/internal/obsreportconfig"
 )
 
+// supported protocols for exporters
+const (
+	// ProtocolProtobufHTTP is the OTLP/HTTP with Protobuf encoding protocol
+	ProtocolProtobufHTTP = "http/protobuf"
+	// ProtocolProtobufGRPC is the OTLP/gRPC protocol
+	ProtocolProtobufGRPC = "grpc/protobuf"
+)
+
```

Comment:
This is now exposed by this package. I think it makes sense to expose it given it is part of the public API